README.md

```diff
@@ -30,9 +33,13 @@ The current repositories relating to TDR, and their purpose are:
 
 - [tdr-antivirus](https://github.com/nationalarchives/tdr-antivirus/) - Task for scanning uploaded files for malware
 - [tdr-api-update](https://github.com/nationalarchives/tdr-api-update/) - Task for sending the results of file checks to the API
+- [tdr-backend-checks-utils](https://github.com/nationalarchives/tdr-backend-checks-utils) - Reads/Writes step function states to S3 to track file checks during processing
+- [tdr-backend-checks-results](https://github.com/nationalarchives/tdr-backend-checks-results) - Aggregates the results of each file check and writes to the S3 in a format readable by other file checks
 - [tdr-checksum](https://github.com/nationalarchives/tdr-checksum/) - Task for calculating the server-side checksum of uploaded files
-- [tdr-download-files](https://github.com/nationalarchives/tdr-download-files/) - Task for transferring a file from S3 to EFS, where it can be used by the rest of the file checks
+- [tdr-file-upload-data](https://github.com/nationalarchives/tdr-file-upload-data/) - Task for retrieving file data to be used by the rest of the file checks
 - [tdr-file-format](https://github.com/nationalarchives/tdr-file-format/) - Task for determining the format of uploaded files
+- [tdr-redacted-files](https://github.com/nationalarchives/tdr-redacted-files) - Task for checking redacted files include original version in upload
+- [tdr-statuses](https://github.com/nationalarchives/tdr-statuses) - Lambda process to update task statuses
```

Comment:
This is slightly pedantic but the tdr-statuses lambda doesn't really update the statuses, it just generates them and passes them on to API update.